index.html

```diff
@@ -15,8 +15,38 @@ <h3 class="header-narrow-text">ULTIMATE CHALLENGER EDITION</h3>
       <div class="left-column">
         <section class="set-range">
         </section>
-        <form class="gameplay-input">
-        </form>
+        <section class="gameplay-cont">
+          <p class="current-range text"> The Current Range is <span class="underline bold">1</span> to <span class="underline bold">100</span> </p>
+          <form class="gameplay-in put">
+            <p class="bold challenger sub-head"> Challenger 1 </p>
+            <section class="challenger-guess">
+              <span class="name-field-cont">
+                <label for="challenger-one-name" class="text"> Name </label>
+                <input id="challenger-one-name" class="name-input" type="text">
+              </span>
+              <span class="guess-field-cont">
+                <label for="challenger-one-guess" class="text"> Guess </label>
+                <input id="challenger-one-guess" class="guess-input" type="text">
+              </span>
+            </section>
+            <p class="bold challenger sub-head"> Challenger 2 </p>
+            <section class="challenger-guess">
+              <span class="name-field">
+                <label for="challenger-two-name" class="text"> Name </label>
+                <input id="challenger-two-name" class="name-input" type="text">
+              </span>
+              <span class="guess-field-cont">
+                <label for="challenger-two-guess" class="text"> Guess </label>
+                <input id="challenger-two-guess" class="guess-input" type="text">
+              </span>
+            </section>
+            <div class="guess-button-cont">
+              <button class="bold text main-button"> SUBMIT GUESS </button disabled=true>
+              <button class="bold text main-button disabled"> RESET GAME </button disabled>
+              <button class="bold text main-button disabled"> CLEAR FORM </button disabled>
```

Comment:
Are the spaces around the button text being used for padding? If so, might be good to style that in CSS